astropy/units/astrophys.py

```diff
@@ -70,15 +70,16 @@
          doc="Electron mass",
          format={'latex': r'M_{e}', 'unicode': 'Mₑ'})
 # Unified atomic mass unit
-def_unit(['u', 'Da', 'Dalton'], 1.6605387e-27 * si.kg, namespace=_ns,
+def_unit(['u', 'Da', 'Dalton'], _si.u.value * si.kg, namespace=_ns,
          prefixes=True, exclude_prefixes=['a', 'da'],
          doc="Unified atomic mass unit")
 
 ##########################################################################
 # ENERGY
 
-def_unit(['Ry', 'rydberg'], 13.605692 * si.eV, namespace=_ns,
-         prefixes=True,
+def_unit(['Ry', 'rydberg'],
+         _si.Ryd.value * _si.c.value * _si.h.value / _si.e.value * si.eV,
+         namespace=_ns, prefixes=True,
```

Comment:
When I put in this calculation I get:
```python
In [5]: Ryd.value * c.value * h.value / e.value * eV
Out[5]: <Quantity 13.605692518464949 eV>
```
But according to http://www.physics.nist.gov/cgi-bin/cuu/Value?rydhcev, this should be `13.605693009(84) eV` so we should probably just hard-code that in the constants module.  If we do that then I'm all for this.